application/models/user/user.php

```diff
@@ -0,0 +1,375 @@
+    $this->$password = $password;
+
+    // The md5 functions is being used to encrypt the password inside database
+    $sql="SELECT Uid from Users WHERE Email='".$username."' and Password='".md5($password)."'";
+    $output=mysqli_query($this->link,$sql);
+    $data=mysqli_fetch_array($output);
+    $row_count=$output->num_rows;
+
+    if($row_count==1){
+      $_SESSION['logged_in']=true;
+      $_SESSION['User_id']=$data['Uid'];
+      return true;
+    }
+    else {
+      return false;
+    }
+
+}
+
+
+  /*
+    Regular Expression: $\S*(?=\S{8,})(?=\S*[a-z])(?=\S*[A-Z])(?=\S*[\d])(?=\S*[\W])\S*$
+    $ = beginning of string
+    \S* = any set of characters
+    (?=\S{8,}) = of at least length 8
+    (?=\S*[a-z]) = containing at least one lowercase letter
+    (?=\S*[A-Z]) = and at least one uppercase letter
+    (?=\S*[\d]) = and at least one number
+    (?=\S*[\W]) = and at least a special character (non-word characters)
+    $ = end of the string
+ */
+  /**
+   * This function will be used to validate the password entered by user
+   *
+   * @param mixed $password
+   *
+   * @return boolean
+   *
+   */
+
+  public function Validate_Password($password){
+
+    if (!preg_match_all('$\S*(?=\S{8,})(?=\S*[a-z])(?=\S*[A-Z])(?=\S*[\d])(?=\S*[\W])\S*$', $password)) {
+      return FALSE;
+    }
+    else {
+      return TRUE;
+    }
+
+  }
+
+  /**
+   *
+   * This function is being used to fetch data from the field
+   *
+   * @param mixed $email
+   *
+   * @return string
+   *
+   */
+  public function Get_Name($email){
+    $sql="SELECT Username from Users WHERE Email='".$email."'";
+      $output=mysqli_query($this->link,$sql);
+      $data=mysqli_fetch_array($output);
+      return $data[0];
+
+  }
+
+
+  /**
+   * @var string $message
+   * This string will return the error messages related to validation of image
+   */
+
+  public string $message;
+
+  /**
+   *
+   * This function will be used to validate the image
+   * @param mixed $imagename
+   * @param mixed $imagesize
+   * @param mixed $imagetype
+   *
+   * @return boolean
+   */
+  public function Validate_Image($imagename,$imagesize,$imagetype){
+      //If there's no image name that means that no image was uploaded so displaying error
+      if (!$imagename) {
+        $this->message = "No image was uploaded";
+        return false;
+      }
+      //If image size is greater than 6MB
+      else if ($imagesize > 6000000) {
+        $this->message = "File size too large";
+          return false;
+      }
+
+      elseif($imagetype != 'image/jpg' && $imagetype !='image/png' && $imagetype!= 'image/jpeg'){
+        $this->message = "Invalid file type";
+        return false;
+      }
+      else{
+        return true;
+      }
+
+  }
+
+  /**
+   *
+   * This function is used to store the image file location in database
+   *
+   * @param mixed $filepath
+   * @param mixed $email
+   *
+   * @return boolean
+   */
+  public function Upload_Image($filepath,$email){
```

Comment:
Please add space after `,` and also do the same changes below of this page where it is required.